### PR TITLE
Added toast component.

### PR DESCRIPTION
**Pull request by j24m**
hello I created toast component, kindly review it.

### CODE REVIEW

File: Components/toast.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
+.toast-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 1rem;
+    font-size: 1.8rem;
+    width: 40rem;
```
Comment (samyakshah3008) [inline Components/toast.css]: Can give max or min-width for responsiveness perspective, can refactor later on

File: Components/toast.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    padding: 1.5rem;
+    border-radius: 0.5rem;
+    position: fixed;
+    bottom: 6rem;
+    animation:fadein 0.5s ;
+}
+/* fadeout 0.5s 0.2s */
```
Comment (samyakshah3008) [inline Components/toast.css]: Can remove this commented code

File: Components/toast.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  /* @-webkit-keyframes fadeout {
+    from {bottom: 6rem; opacity: 1;}
+    to {bottom: 0; opacity: 0;}
+  }
+  
+  @keyframes fadeout {
+    from {bottom: 6rem; opacity: 1;}
+    to {bottom: 0; opacity: 0;}
+  } */
+
```
Comment (samyakshah3008) [inline Components/toast.css]: Remove this as part of refactoring

File: Components/toast.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+            <div class="btn-toastShow">
+                <button id="btn-toast-left" class="btn btn-primary">Toast-left</button>
+                <button id="btn-toast-center" class="btn btn-primary">Toast-right</button>
+                <button id="btn-toast-right" class="btn btn-primary">Toast-center</button>
+            </div>
+            <!-- buttons to show toast end here -->
+            
+            <!-- toast left starts here -->
+            <div id="toast-body-left" class="toast-container toast-left">
+                <div>Can't send photo.</div>
```
Comment (samyakshah3008) [inline Components/toast.html]: This comments are helpful for smooth reviewing

**Review by samyakshah3008 — APPROVED** [review]
Clean code with good use of comments 
Added few suggestions in single comments 
You can also attach live link or add more preview description with image for better review 

Rest is all good for approval ✅

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks great. Good to merge.

File: Components/toast.css (hunk 2 above)
Comment (j24m) [reply Components/toast.css]: done.

**Review by mehulsatardekar — APPROVED** [review]
everything looks good and ready to merge this branch 👍

**Pull request merged**